<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AnhThi-HoaiThuong-PhuocThuan_MIDTERM</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>661</width>
      <height>721</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
    background-color: rgb(56,58,89);
  border-radius: 20px;

}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <widget class="QLabel" name="iframe_old">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>400</width>
       <height>341</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>400</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>180</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rockwell Extra Bold</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; color:#ff00ff;&quot;&gt;BEFORE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="iframe_new">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>360</y>
       <width>401</width>
       <height>351</height>
      </rect>
     </property>
     <property name="toolTipDuration">
      <number>-1</number>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>460</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rockwell Extra Bold</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#ff00ff;&quot;&gt;AFTER&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelAfter">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>540</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMULTIPLY_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>330</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>SAVE IMAGE</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>251</width>
      <height>721</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
    background-color:rgb(0, 0, 127);
    border-radius: 20px;

}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rockwell Extra Bold</family>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-style:normal; text-decoration: underline; color:#5500ff;&quot;&gt;NHÃ“M TTT&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>231</width>
       <height>501</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QScrollArea{
border-radius: 30px;
}</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>231</width>
        <height>501</height>
       </rect>
      </property>
      <widget class="QPushButton" name="btnLinear">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>550</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Linear</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnMULTIPLY">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>10</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: blue;
background-color: orange;</string>
       </property>
       <property name="text">
        <string>MULTIPLY</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSUBTRACT">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: blue;
background-color: orange;</string>
       </property>
       <property name="text">
        <string>SUBTRACT</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnGRAMMA">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>40</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: blue;
background-color: orange;</string>
       </property>
       <property name="text">
        <string>GRAMMA</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSUBTRACT_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>10</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
  color: blue;
background-color: orange;</string>
       </property>
       <property name="text">
        <string>ADD</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600; color:#ffffff;&quot;&gt;BASIC&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnBlur">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>70</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: blue;
background-color: orange;</string>
       </property>
       <property name="text">
        <string>Blur</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnGauss">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>70</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: blue;
background-color: orange;</string>
       </property>
       <property name="text">
        <string>Gauss</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnMedian">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>100</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: blue;
background-color: orange;</string>
       </property>
       <property name="text">
        <string>Median</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSharpen">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>160</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: blue;
background-color: orange;</string>
       </property>
       <property name="text">
        <string>Sharpen</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnMean">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>100</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: blue;
background-color: orange;</string>
       </property>
       <property name="text">
        <string>Mean</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSharpen_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>160</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: blue;
background-color: orange;</string>
       </property>
       <property name="text">
        <string>Linear</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>110</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600; color:#ffffff;&quot;&gt;SMOO-&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>129</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600; color:#ffffff;&quot;&gt;THING&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnBilateral">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>130</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: blue;
background-color: orange;</string>
       </property>
       <property name="text">
        <string>Bilateral</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSharpen_3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>130</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: blue;
background-color: orange;</string>
       </property>
       <property name="text">
        <string>Filter2D</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>200</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt; font-weight:600; color:#ffffff;&quot;&gt;GRADIENT&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSobel">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>190</y>
         <width>71</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: blue;
background-color: orange;</string>
       </property>
       <property name="text">
        <string>Sobel</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnPrewitt">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>190</y>
         <width>71</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: blue;
background-color: orange;</string>
       </property>
       <property name="text">
        <string>Prewitt</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnLaplacian">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>220</y>
         <width>71</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: blue;
background-color: orange;</string>
       </property>
       <property name="text">
        <string>Laplacian</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>270</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt; font-weight:600; color:#ffffff;&quot;&gt;CANNY&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnGy">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>250</y>
         <width>81</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: blue;
background-color: orange;</string>
       </property>
       <property name="text">
        <string>Gy</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnGx">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>250</y>
         <width>81</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: blue;
background-color: orange;</string>
       </property>
       <property name="text">
        <string>Gx</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnGxAddGy">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>280</y>
         <width>81</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: blue;
background-color: orange;</string>
       </property>
       <property name="text">
        <string>|Gx|+|Gy|</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnGxDivGy">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>280</y>
         <width>81</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: blue;
background-color: orange;</string>
       </property>
       <property name="text">
        <string>Gx/Gy</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>320</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt; font-weight:600; color:#ffffff;&quot;&gt;THRESH-&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>340</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt; font-weight:600; color:#ffffff;&quot;&gt;HOLDING&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnGAUSSIAN_C">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>310</y>
         <width>81</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: blue;
background-color: orange;</string>
       </property>
       <property name="text">
        <string>GAUSSIAN_C</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnMEAN_C">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>340</y>
         <width>81</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: blue;
background-color: orange;</string>
       </property>
       <property name="text">
        <string>MEAN_C</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnTRUNC">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>340</y>
         <width>81</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: blue;
background-color: orange;</string>
       </property>
       <property name="text">
        <string>TRUNC</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnTOZERO">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>310</y>
         <width>81</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: blue;
background-color: orange;</string>
       </property>
       <property name="text">
        <string>TOZERO</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnOTSU">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>370</y>
         <width>81</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: blue;
background-color: orange;</string>
       </property>
       <property name="text">
        <string>OTSU</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnBINARY">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>370</y>
         <width>81</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: blue;
background-color: orange;</string>
       </property>
       <property name="text">
        <string>BINARY</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnDrectional">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>220</y>
         <width>71</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: blue;
background-color: orange;</string>
       </property>
       <property name="text">
        <string>Drectional</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnMULTIPLY_5">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>400</y>
         <width>81</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: blue;
background-color: orange;</string>
       </property>
       <property name="text">
        <string>K-MEAN</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rockwell Extra Bold</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; color:#ffff00;&quot;&gt;PHOTO&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Calligraphy</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; color:#ffffff;&quot;&gt;Size of filter&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="boxFilter">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>170</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-9</number>
     </property>
     <property name="singleStep">
      <number>2</number>
     </property>
     <property name="value">
      <number>3</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="pixmap">
      <pixmap>app.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rockwell Extra Bold</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; color:#ffff00;&quot;&gt;WONDER&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>911</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="test.qrc"/>
 </resources>
 <connections/>
</ui>
